subsys/nrf_security/src/drivers/cracen/cracenpsa/src/ctr_drbg.c

```diff
@@ -82,7 +82,8 @@ static psa_status_t ctr_drbg_update(uint8_t *data)
 {
 	psa_status_t status = SX_OK;
 
-	char temp[CRACEN_PRNG_ENTROPY_SIZE + CRACEN_PRNG_NONCE_SIZE];
+	char temp[CRACEN_PRNG_ENTROPY_SIZE + CRACEN_PRNG_NONCE_SIZE] __aligned(
```

Comment:
Do we not need to ifdef this and the next one with CONFIG_DCACHE?

Reply:
It compiles fine with DCACHE disable, Kconfig gives 32 as "cache line size" for that case. I thought it looks nicer without the extra defines, but I can add something if you really want it :)

Reply:
Please add your first sentence as a comment so that we know what's happening, then it is fine to keep it for sure!